tests/Tokenizer/TokensTest.php

```diff
@@ -739,6 +743,7 @@ public function testFindBlockEndInvalidType(): void
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessageMatches('/^Invalid param type: "-1"\.$/');
 
+        // @phpstan-ignore-next-line
         $tokens->findBlockEnd(-1, 0);
```

Comment:
@keradus This is a place where `Tokens::BLOCK_TYPE_*` type actually helps to find invalid usage of this method.